nessie/src/main/java/org/apache/iceberg/nessie/UpdateableReference.java

```diff
@@ -62,7 +62,7 @@ public Reference getReference() {
 
   public void checkMutable() {
     Preconditions.checkArgument(
-        mutable, "You can only mutate tables when using a branch without a hash or timestamp.");
+        mutable, "You can only mutate contents when using a branch without a hash or timestamp.");
```

Comment:
nit: should this say `content` maybe?

Reply:
I think it can be plural as mentioned here.
https://github.com/apache/iceberg/pull/8909#discussion_r1375260224